runtime/core/decoder/torch_asr_decoder.cc

```diff
@@ -272,6 +277,18 @@ void TorchAsrDecoder::UpdateResult() {
   }
 }
 
+float TorchAsrDecoder::AttentionDecoderScore(const torch::Tensor& prob,
+                                             const std::vector<int>& hyp,
+                                             const int eos) {
```

Comment:
const is not required here.

Reply:
CumulateScore is better than AttentionDecoderScore

Reply:
CumulateScore may be hard to understand